src/webgpu/api/validation/queue/copyImageBitmapToTexture.spec.ts

```diff
@@ -0,0 +1,409 @@
+export const description = `
+copyImageBitmapToTexture Validation Tests in Queue.
+
+Test Coverage:
+- For source.imagBitmap:
+  - imageBitmap generated from ImageData:
+    - Check that an error is generated when imageBitmap is closed. Otherwise, no error should be
+      generated.
+
+- For destination.texture:
+  - For 2d destination textures:
+    - Check that an error is generated when texture is in destroyed state. Otherwise, no error should
+      be generated.
+    - Check that an error is generated when texture is created without usage COPY_DST. Otherwise,
+      no error should be generated.
+    - Check that an error is generated when sample count is not 1. Otherwise, no error should be
+      generated.
+    - Check that an error is generated when mipLevel is too large. Otherwise, no error should be
+      generated.
+    - Check that an error is generated when texture format is not valid. Otherwise, no error should
+      be generated.
+
+- For copySize:
+  - Check that an error is generated when copySize has 0 in width, height or depth. Otherwise,
+    no error should be generated.
+  - Check that an error is generated when destination.texture.origin + copySize is too large.
+    Otherwise, no error should be generated.
+
+TODO: more test to cover source.imageBitmap generated from different source.
+TODO: 1d and 3d texture
```

Comment:
I had a thought reading these tests. The usefulness of copyImageBitmapToTexture for 1d and 3d textures is very limited. I think we should consider disallowing them in the WebGPU spec for MVP, with possible addition later on. ImageBitmap interaction is one of the more complicated parts of an implementation (since it interacts with other parts of the Web platform) so it would be best to avoid extra complexity with minimal benefit. Would you file a gpuweb issue about this?

Reply:
Agree. I'll file one.

Reply:
Softly ping @kainino0x and cc @Kangz

Reply:
Either way seems fine, but I'd rather have a polished version of 2D copyImageBitmapToTexture for the MVP than something that does 1D / 3D but isn't as nice. Also the 0 copy path investigation is probably more important than support for 1D and 3D in copyIB2T.

Reply:
Ack.